openwhisk/01-dice-color.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Those .json files are examples, isnt it ? By convention we movet them to the examples/ directory

Reply:
@chmouel yes, had thought to move them, but it took so much time to get the instructions working that I had hoped to defer this to a subsequent PR as it seemed the priority was to get the resources themselves updated to v1beta1 level.   If this is a "blocker" I can try to find some time in the next several days to do so, but would it suffice to open an issue and assign it to myself against just this catalog entry?

Reply:
@chmouel I decided to co-locate the JSON parameter (input) files in the same subdirectory with the Java pipelineRun YAML as this is the only pipelinerun (example) that uses these input files.